R/arg.R

```diff
@@ -1,85 +1,59 @@
-#' Find the expression associated with an argument
+#' Introspect an argument
 #'
-#' \code{arg_expr()} finds the full expression; \code{arg_text()} turns the
-#' expression into a single string; \code{arg_label()} formats it nicely for
-#' use in messages. \code{arg_env()} finds the environment associated with
-#' the expression.
+#' \code{arg_info()} and \code{arg_info_()} are the workhorses of all
+#' functions providing information about the evaluation context of an
+#' argument (e.g., \code{\link{arg_expr}()}, \code{\link{arg_env}()},
+#' etc). These functions climb the call stack to find where an
+#' argument was first supplied, with which expression, in which
+#' evaluation environment.
 #'
-#' These functions never force promises, and will work even if a promise has
-#' previously been forced.
+#' \code{arg_info_()} is the standard-evaluation version of
+#' \code{arg_info()} and takes a symbol and a call stack object.
 #'
-#' @param x A promise (function argument)
-#' @export
-#' @examples
-#' # Unlike substitute(), arg_expr() finds the original expression
-#' f <- function(x) g(x)
-#' g <- function(y) h(y)
-#' h <- function(z) list(substitute(z), arg_expr(z))
+#' @param x An argument of a function.
+#' @return A list containing:
+#' \itemize{
```

Comment:
From R-exts

> Description of the function’s return value.
> 
> If a list with multiple values is returned, you can use entries of the form
>
>\item{comp_i}{Description of comp_i.}
>for each component of the list returned. Optional text may precede93 this list (see for example the help for rle). >Note that \value is implicitly a \describe environment, so that environment should not be used for listing >components, just individual \item{}{} entries.

So you can drop the `\itemize`, and use the `\item{expr}{...}` form